crates/hir/src/lib.rs

```diff
@@ -3691,7 +3695,7 @@ pub enum CaptureKind {
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 pub struct Type {
     env: Arc<TraitEnvironment>,
-    ty: Ty,
+    pub ty: Ty,
```

Comment:
This should stay private, we don't want to leak hir-ty `Ty`pes into the ide layer

Reply:
removed!